docs/install-w-machine.md

```diff
@@ -1,231 +1,173 @@
 <!--[metadata]>
 +++
-title = "Install and use Swarm"
-description = "Swarm release notes"
-keywords = ["docker, swarm, clustering, discovery, release,  notes"]
+title = "Getting started with Docker Swarm"
+description = "Introduction to Swarm commands and concepts"
+keywords = ["docker, swarm, clustering, discovery, examples"]
 [menu.main]
 parent="workw_swarm"
 +++
 <![end-metadata]-->
 
-# Install and Create a Docker Swarm
+# Swarm get started (novice)
 
-You use Docker Swarm to host and schedule a cluster of Docker containers. This section introduces you to Docker Swarm by teaching you how to create a swarm
-on your local machine using Docker Machine and VirtualBox.
+## Introduction
 
-## Prerequisites
+This getting started example shows you how to create a Docker Swarm, the
+native clustering tool for Docker.
 
-Make sure your local system has VirtualBox installed. If you are using Mac OS X
-or Windows and have installed Docker, you should have VirtualBox already
-installed.
+When you finish, you'll have a Docker Swarm up and running in VirtualBox on your
+local Mac or Windows computer. You can use this Swarm as personal development
+sandbox.
 
-Using the instructions appropriate to your system architecture, [install Docker
-Machine](http://docs.docker.com/machine/install-machine).
+To use Docker Swarm on Linux, see [Create a Swarm for
+development](https://docs.docker.com/swarm/install-manual/).
 
-## Create a Docker Swarm
+## Install Docker Toolbox
 
-Docker Machine gets hosts ready to run Docker containers. Each node in your
-Docker Swarm must have access to Docker to pull images and run them in
-containers. Docker Machine manages all this provisioning for your swarm.
+Download and install [https://www.docker.com/docker-toolbox](Docker Toolbox).
 
-Before you create a swarm with `docker-machine`, you associate each
-node with a discovery service. This example uses the token discovery
-service hosted by Docker Hub (only for testing/dev, not for production).
-This discovery service associates a token with instances of the Docker
-Daemon running on each node. Other discovery service backends such as
-`etcd`, `consul`, and `zookeeper` are [available](discovery.md).
+The toolbox installs a handful of tools on your local Windows or Mac OS X computer. In this exercise, you use three of those tools:
 
-1. List the machines on your system.
+ - Docker Machine: To deploy virtual machines that run Docker Engine.
+ - VirtualBox: To host the virtual machines deployed from Docker Machine.
+ - Docker Client: To connect from your local computer to the Docker Engines on the VMs and issue docker commands to create the Swarm.
+
+The following sections provide more information on each of these tools. The rest of the document uses the abbreviation, VM, for virtual machine.
+
+## Create three VMs running Docker Engine
+
+1. Open a terminal on your computer. Use Docker Machine to list any VMs in VirtualBox.
```

Comment:
@rolfedh these are actually two steps one of which as a secondary procedure.

```
1. Open a shell on your Mac or Windows computer.
2.  (Optional) Conserve resources by stopping any VMs you may have running.
      a. Use Docker Machine to list any currently running VMs.
          (your command example)
      b. Stop the running VMs:
           (your command example)
```

BTW, in general the rule is, the single command on the command line.  Any follow-text or explainer on a new indented line so, as an example:

```
2. Optional: To conserve system resources, stop any virtual machines you are not using. For example, to stop the VM named `default`, enter:
```

Becomes

```
2. (Optional)To conserve system resources, stop any virtual machines you are not using. 
  For example, to stop the VM named `default`:
```

We haven't used enter to introduce the command line a colon should work.